cura/Settings/MaterialSettingsVisibilityHandler.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2016 Ultimaker B.V.
+# Uranium is released under the terms of the AGPLv3 or higher.
+
+import UM.Settings.Models
+
+class MaterialSettingsVisibilityHandler(UM.Settings.Models.SettingVisibilityHandler):
+    def __init__(self, parent = None, *args, **kwargs):
+        super().__init__(parent = parent, *args, **kwargs)
+
+        material_settings = set([
+            "material_print_temperature",
+            "material_bed_temperature",
+            "material_standby_temperature",
+            "cool_fan_speed",
+            "retraction_amount",
+            "retraction_speed",
+        ])
```

Comment:
This is sort of double data. We have a different set of visible settings in CuraApplications, and as I now understand it this will make these six additional settings visible at every launch (I may be wrong on this, maybe it's every time you start the material manager?).

If you really want a separate set of visible settings in the material manager, then I suggest that you at least make a setting for it and define that setting here. The setting doesn't necessarily need to be modifiable from the interface.